api/db/changelog/db.changelog-124-add-demographic-gender-identity.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For safety, could we run this Liquibase changeset first and then merge the code changes?

Reply:
This is in your other PR right? So you don't need it here anymore after rebasing that one I guess.

Reply:
I am just waiting for this PR to be approved, the plan is 
to merge the other PR in 
remove this (this is here for context) and 
merge this  PR then